dev/devicelab/bin/tasks/flutter_attach_test_android.dart

```diff
@@ -53,7 +53,11 @@ Future<void> testReload(Process process, { Future<void> Function() onListening }
   process.exitCode.then<void>((int processExitCode) { exitCode = processExitCode; });
 
   Future<dynamic> eventOrExit(Future<void> event) {
-    return Future.any<dynamic>(<Future<dynamic>>[ event, process.exitCode ]);
+    return Future.any<dynamic>(<Future<dynamic>>[
+      event,
+      process.exitCode,
+      Future<void>.delayed(const Duration(seconds: 10)),
```

Comment:
Keep the test from running for 15 minutes if it gets stuck

Reply:
This would be a great comment in the code.

Reply:
Ahh yeah, good idea :)